build.gradle

```diff
@@ -638,7 +638,7 @@ platform {
 	}
 	bnd group: 'ru.yandex.qatools.allure', name: 'allure-java-adaptor-api', {
 		// tika exported package version does not seem to resemble bundle version
-		instruction 'Import-Package', 'org.apache.tika.*;version="[1.0.0,2.0.0)",*'
+		instruction 'Import-Package', 'org.apache.tika.*;version="[1.28.4,2.9.1)",*'
```

Comment:
This only impacts the `Import-Package` instructions, not the dependency version used.

Reply:
I see, but I cannot find any other place where tika has been used.